### PR TITLE
Add support for obtaining the package for the crate that the manifest represents

### PR DESCRIPTION
**Pull request by kellpossible**
Support for #11

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -297,6 +297,55 @@ impl Manifest {
...
+    /// 
+    /// # Examples
+    /// 
+    /// ```rust
+    /// use find_crate::Manifest;
+    /// use proc_macro2::{Ident, Span, TokenStream};
+    /// use quote::quote;
+    ///
+    /// fn crate_name() -> TokenStream {
+    /// }
```
Comment (kellpossible) [inline src/lib.rs:315]: Need to finish example, after I confirm that it would work as expected

**Comment by kellpossible** [comment]
@taiki-e I have a nice feature in one of my crates which is waiting on this functionality, if you get a chance to review, would be fantastic!

**Review by taiki-e — APPROVED** [review]
LGTM, thanks!

**Comment by taiki-e** [comment]
Published in 0.6.0.

**Comment by kellpossible** [comment]
Awesome @taiki-e thanks a lot!

**Pull request merged**